Sources/SwiftyMarkdown/SwiftyMarkdown.swift

```diff
@@ -21,6 +21,7 @@ public enum CharacterStyle : CharacterStyling {
 	case none
 	case bold
 	case italic
+    case boldItalic
```

Comment:
Nit: indent

Reply:
Curses! They use tabs for indent, and we use spaces!! Looks like it also affected the last PR. I guess I'll stick with tabs?

Reply:
Looks like they're already inconsistent even within the same file, so I just went with spaces for the files I touched.